weave-js/src/components/Tag/Alias.tsx

```diff
@@ -1,10 +1,10 @@
+import {Icon} from '@wandb/weave/components/Icon';
```

Comment:
If you haven't, could you double-check that this works in Storybook? I seem to recall issues with how it resolved `@wandb`.